neurobooth_os/server_acq.py

```diff
@@ -48,6 +49,11 @@ def Main():
         raise
 
 
+def is_camera(stream_name: str) -> bool:
+    """Test to see if a stream is a camera stream based on its name."""
+    return stream_name.split("_")[0] in ["FLIR", "Intel", "IPhone"]
```

Comment:
probably should be configurable. Maybe when we work on task/device config.

Reply:
Agreed, intent was to address in rework. Currently is just a temporary refactor for the several places we were doing this.